js/markerjs2.js

```diff
@@ -13,4 +13,4 @@
-//# sourceMappingURL=markerjs2.js.map
+// sourceMappingURL=markerjs2.js.map
```

Comment:
do we need this line at all? I don't think we have the source map here

Reply:
I was not sure since I'm not familiar with it. I removed it and nothing broke so we should be fine.